src/net.cpp

```diff
@@ -170,18 +167,17 @@ static int GetnScore(const CService& addr)
 }
 
 // Is our peer's addrLocal potentially useful as an external IP source?
-bool IsPeerAddrLocalGood(CNode *pnode)
+bool IsPeerAddrLocalGood(CNode* pnode)
 {
     CService addrLocal = pnode->GetAddrLocal();
     return fDiscover && pnode->addr.IsRoutable() && addrLocal.IsRoutable() &&
            !IsLimited(addrLocal.GetNetwork());
 }
 
 // pushes our own address to a peer
-void AdvertiseLocal(CNode *pnode)
+void AdvertiseLocal(CNode* pnode)
 {
-    if (fListen && pnode->fSuccessfullyConnected)
-    {
+    if (fListen && pnode->fSuccessfullyConnected) {
```

Comment:
We run https://github.com/bitcoin/bitcoin/tree/master/contrib/devtools#clang-format-diffpy only on the diff and not the whole file.

Reply:
Hi MarkoFalke. I am afraid, appveyor will reject a file that is not fully formatted according to clang rules. Better bite the bullet now?

Reply:
We don't check formatting in appveyor. Do you have a failing build to link to?